### PR TITLE
Added the second color sensor to index

### PR DESCRIPTION
**Pull request by eddiert1239**
added the second color sensor to index just by adding to the already existing ingest sensor code

### CODE REVIEW

File: src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java [hunk 1]
```diff
@@ -196,12 +203,14 @@ public boolean isFeederMotorOn() {
...
 
     /**
-     * Checks if ingest has the correct cargo
+     * Checks if ingest has the correct cargo (also includes the top sensor)
      */
     @Log(name = "Ingest Cargo")
     public boolean ingestHasCorrectCargo() {
         return ((teamColor == Alliance.Blue && ingestBlueColor.get())
-                || teamColor == Alliance.Red && ingestRedColor.get());
+                || teamColor == Alliance.Red && ingestRedColor.get()
```
Comment (bobtseattle) [inline src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java:211]: Simpler to read and understand as
(teamColor == Alliance.Blue && ( ingestBlueColor.get() || topBlueColor.get()) ....

**Review by bobtseattle — COMMENTED** [review]
Rename to ingest top and simplify the logic.

File: src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java (hunk 1 above)
Comment (eddiert1239) [reply src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java:211]: I see. Thanks

**Review by bobtseattle — APPROVED** [review]
Will need arduino code updates, the sensor to be wired, and the new wires from the arduino to the roborio.

**Review by jamesdooley4 — APPROVED** [review]

**Pull request merged**